models/cow_protocol/cow_protocol_trades.sql

```diff
@@ -0,0 +1,38 @@
+{{ config(
+        alias ='trades'
+        )
+}}
+SELECT *
+FROM
+(
+        SELECT
+            'ethereum' AS blockchain,
+            'CoW Protocol' AS project,
+            '1' AS version,
+            TRY_CAST(date_trunc('DAY', block_time) AS date) AS block_date,
+            block_time,
+            buy_token AS token_bought_symbol,
+            sell_token AS token_sold_symbol,
+            token_pair,
+            units_bought AS token_bought_amount,
+            units_sold AS token_sold_amount,
+            atoms_bought AS token_bought_amount_raw,
+            atoms_sold AS token_sold_amount_raw,
+            usd_value AS amount_usd,
+            buy_token_address AS token_bought_address,
+            sell_token_address AS token_sold_address,
+            trader AS taker,
+            '' AS maker,
+            project_contract_address,
+            tx_hash,
+            trader AS tx_from,
+            receiver AS tx_to,
```

Comment:
@bh2smith Also please double check if these are correct as well.

Reply:
That is a very interesting question. Technically I would think `tx_from` is actually the solver address (if one wanted to compare / join on `ethrereum.transactions`). I kinda wish these column names were trade from and trade to (if that is what they are intended to mean). Do we still have `trader_a` and `trader_b` because this would be redundant and I would actually propose the following:

```suggestion
            solver AS tx_from,
            <SETTLEMENT_CONTRACT_ADDRESS> AS tx_to,
```

Note that the `SETTLEMENT_CONTRACT_ADDRESS` shouldn't need to be hardcoded as indicated here (since there are two versions of this deployed, we can actually use `tx.from` and `tx.to` from `ethereum.transactions` table for this.